diff --git '${BUILD_DIR}'at91bootstrap3-v3.8.9/board/sama5d2_xplained/sama5d2_xplained.c '${BUILD_DIR}'at91bootstrap3-v3.8.9/board/sama5d2_xplained/sama5d2_xplained.c
index ac08fd9..c2720d7 100644
--- '${BUILD_DIR}'at91bootstrap3-v3.8.9/board/sama5d2_xplained/sama5d2_xplained.c
+++ '${BUILD_DIR}'at91bootstrap3-v3.8.9/board/sama5d2_xplained/sama5d2_xplained.c
@@ -50,14 +50,14 @@
 
 static void at91_dbgu_hw_init(void)
 {
-	const struct pio_desc dbgu_pins[] = {
+/*	const struct pio_desc dbgu_pins[] = {
 		{"RXD1", CONFIG_SYS_DBGU_RXD_PIN, 0, PIO_DEFAULT, PIO_PERIPH_A},
 		{"TXD1", CONFIG_SYS_DBGU_TXD_PIN, 0, PIO_DEFAULT, PIO_PERIPH_A},
 		{(char *)0, 0, 0, PIO_DEFAULT, PIO_PERIPH_A},
 	};
 
 	pio_configure(dbgu_pins);
-	pmc_sam9x5_enable_periph_clk(CONFIG_SYS_DBGU_ID);
+	pmc_sam9x5_enable_periph_clk(CONFIG_SYS_DBGU_ID);*/
 }
 
 static void initialize_dbgu(void)
